### PR TITLE
SEO-2024 Blazor DateTime Picker External Linking Changes

### PR DESCRIPTION
**Pull request by Yvone-Atieno**
Hi @ChristoIssac ,

Please review the changer for Blazor DateTime Picker External Linking Changes.

Regards,
Yvone.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,6 @@
...
 
-A quick start blazor server app that allows you to configure the Blazor DateTimePicker component of Syncfusion with its available options. This project contains simple code customizations, as well as some important features such as setting specific date time value, formatting and range restriction for date time selection.
-Documentation: https://blazor.syncfusion.com/documentation/datetime-picker/getting-started
-
-Online examples: https://blazor.syncfusion.com/demos/datetime-picker/default-functionalities
+[Blazor DateTime Picker](https://www.syncfusion.com/blazor-components/blazor-datetime-picker?utm_source=github&utm_medium=listing&utm_campaign=blazor-buttons-github-samples) is a lightweight and mobile-friendly component that allows end users to enter or select date and time values from a pop-up calendar and drop-down time list. A quick start blazor server app that allows you to configure the Blazor DateTimePicker component of Syncfusion with its available options. This project contains simple code customizations, as well as some important features such as setting specific date time value, formatting and range restriction for date time selection.
```
Comment (Girija-Rajendran) [inline README.md]: invalid UTM param

Comment (Yvone-Atieno) [reply README.md]: resolved

File: README.md [hunk 2]
```diff
@@ -1,9 +1,6 @@
...
 
-A quick start blazor server app that allows you to configure the Blazor DateTimePicker component of Syncfusion with its available options. This project contains simple code customizations, as well as some important features such as setting specific date time value, formatting and range restriction for date time selection.
-Documentation: https://blazor.syncfusion.com/documentation/datetime-picker/getting-started
-
-Online examples: https://blazor.syncfusion.com/demos/datetime-picker/default-functionalities
+[Blazor DateTime Picker](https://www.syncfusion.com/blazor-components/blazor-datetime-picker?utm_source=github&utm_medium=listing&utm_campaign=blazor-calendars-github-samples) is a lightweight and mobile-friendly component that allows end users to enter or select date and time values from a pop-up calendar and drop-down time list. A quick start blazor server app that allows you to configure the Blazor DateTimePicker component of Syncfusion with its available options. This project contains simple code customizations, as well as some important features such as setting specific date time value, formatting and range restriction for date time selection.
```
Comment (Girija-Rajendran) [inline README.md]: Is this correct?
image
image

Check this in all places

**Comment by Yvone-Atieno** [comment]
resolved

**Review by Girija-Rajendran — APPROVED** [review]

**Pull request merged**